Macropad_Hotkeys/code.py

```diff
@@ -48,51 +49,50 @@ def switch(self):
 
 
 # INITIALIZATION -----------------------
-
-macropad = MacroPad()
-macropad.display.auto_refresh = False
-macropad.pixels.auto_write = False
-
-# Set up displayio group with all the labels
-group = displayio.Group()
-for key_index in range(12):
-    x = key_index % 3
-    y = key_index // 3
-    group.append(label.Label(terminalio.FONT, text='', color=0xFFFFFF,
-                             anchored_position=((macropad.display.width - 1) * x / 2,
-                                                macropad.display.height - 1 -
-                                                (3 - y) * 12),
-                             anchor_point=(x / 2, 1.0)))
-group.append(Rect(0, 0, macropad.display.width, 12, fill=0xFFFFFF))
-group.append(label.Label(terminalio.FONT, text='', color=0x000000,
-                         anchored_position=(macropad.display.width//2, -2),
-                         anchor_point=(0.5, 0.0)))
-macropad.display.show(group)
-
-# Load all the macro key setups from .py files in MACRO_FOLDER
-apps = []
-files = os.listdir(MACRO_FOLDER)
-files.sort()
-for filename in files:
-    if filename.endswith('.py'):
-        try:
-            module = __import__(MACRO_FOLDER + '/' + filename[:-3])
-            apps.append(App(module.app))
-        except (SyntaxError, ImportError, AttributeError, KeyError, NameError,
-                IndexError, TypeError) as err:
+if True:
```

Comment:
The final decision to include this or not isn't up to me, but my two cents: I'm not really in favor of adding the `if True:` for indention in these projects. 

The primary audience for these learn guide projects tends to be folks relatively new to programming with little experience. They are encouraged by the guides to use Mu editor which does not have the code block folding feature that comes with some more advanced IDEs like vscode and PyCharm, so it's less likely that they'll get the benefit from this. And it may lead to a bit of confusion if they end up wondering why the if condition is used which may appear useless to them if they don't know about the more advanced editors and their code folding feature.

Reply:
I’m inclined to agree. This is a good skateboard trick and something I’ll keep in mind for future personal projects…but as it’s a specific editor feature and not a general Python feature, best avoided here.

Reply:
<details><summary>Code folding support is pervasive</summary><P/>

Wikipedia shows that code folding functionality is [quite broadly supported](https://en.wikipedia.org/wiki/Code_folding#Software_with_code_folding_capability) by editors.  In that entire list, only four editors don't support code-folding of Python (either by indentation or by language parsing).

Admittedly, MU does not appear to support code folding.  When Notepad++, VSCode, and both VIM and emacs(!) support a feature, I'd suggest it's an expected capability of code editors.
</details>

But, FoamyGuy has a point.  I've added a comment that should reduce potential for confusion.  PaintYourDragon ... just say the word, and I'll un-indent (but will shed some tears).

Reply:
Rather de-indent, thanks, sorry.